app/routes/circulars/index.tsx

```diff
@@ -204,14 +223,10 @@ export default function () {
         <Link to="/docs/circulars">documentation</Link> for help with
         subscribing to or submitting Circulars.
       </p>
-      <ButtonGroup className="position-sticky top-0 bg-white margin-bottom-1 padding-top-1">
-        <Form
-          className="display-inline-block usa-search usa-search--small"
-          role="search"
-        >
-          <Label srOnly={true} htmlFor="query">
-            Search
-          </Label>
+      <div className="position-sticky top-0 bg-white margin-bottom-1 padding-top-1">
```

Comment:
Why did you change this from a ButtonGroup to a div?

Reply:
I believe this was originally changed after the repositioning of the end of the Form element, as the ButtonGroup didn't work, but I see now that changing it to a div removes the sticky property. I was able to restore it to a ButtonGroup by changing the order of the Form start and where the ButtonGroup starts. changes made in [c206c92](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1104/commits/c206c92cc44cfd49a8657c1ca3df6fa70ebc1f66)